guides/getting_started/quickstart.livemd

```diff
@@ -0,0 +1,124 @@
+# Quickstart
```

Comment:
I think Installation, Introduction, CodeGen and quickstart could all be part of the same file

Reply:
I still kept Codegen separate and condensed the others into the quickstart. Codegen follows a different approach, and I would like to be more detailed with it in its current form.